src/modules/wara/advisor/advisor.psm1

```diff
@@ -247,7 +247,7 @@ class advisorResourceObj : IComparable, IEquatable[object] {
         return $true
     }
 
-    # Implement the CompareTo method to compare two advisorResourceObj objects
+<#     # Implement the CompareTo method to compare two advisorResourceObj objects
```

Comment:
Is this commented out means, at this point, we do not need comparative capability (sorting, etc), right?